app/uk/gov/hmrc/transitmovementsauditing/controllers/AuditController.scala

```diff
@@ -74,6 +78,12 @@ class AuditController @Inject() (
       }
   }
 
+  private def exceedsMessageSize(implicit request: Request[Source[ByteString, _]]): Boolean =
```

Comment:
@drnaylor-oc  is it fine to use ```CONTENT_LENGTH``` instead of StreamWithSize method to calculate the size?

Reply:
No, it's not likely `content-length` will be there given we'll be streaming to auditing, thanks to how streaming works.

Reply:
Yes, content-length probably won't be available to us.